File: app/src/main/java/com/babestudios/companieshouse/data/model/company/Accounts.java
package com.babestudios.companieshouse.data.model.company;

import com.google.gson.annotations.SerializedName;

public class Accounts {

	@SerializedName("accounting_reference_date")
	public AccountingReferenceDate accountingReferenceDate;

	@SerializedName("last_accounts")
	public LastAccounts lastAccounts;

	@SerializedName("next_due")
	public String nextDue;

	public boolean overdue;

	@SerializedName("next_made_up_to")
	public String nextMadeUpTo;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/company/Accounts.kt
package com.babestudios.companyinfouk.data.model.company

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Accounts(
		@SerializedName("accounting_reference_date")
		var accountingReferenceDate: AccountingReferenceDate? = null,

		@SerializedName("last_accounts")
		var lastAccounts: LastAccounts? = null,

		@SerializedName("next_due")
		var nextDue: String? = null,

		var overdue: Boolean = false,

		@SerializedName("next_made_up_to")
		var nextMadeUpTo: String? = null
) : Parcelable
